.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         kubeapiserver-version: [ v1.21.10, v1.22.7, v1.23.4, v1.24.2, v1.25.0, v1.26.0, v1.27.3 ]
-        karmada-version: [ release-1.4, release-1.5, release-1.6 ]
+        karmada-version: [ release-1.4, release-1.5, release-1.6, release-1.7 ]
```

Comment:
What's the policy for this version list?   most recent three minor release?

Reply:
Exactly, the most recent three minor release